Tools-Override/tests.targets

```diff
@@ -232,9 +232,15 @@
       <IlcInputFolderContents Include="$(ILCFXInputFolder)/*" />
       <TestCommandLines Include="mklink /H %(IlcInputFolderContents.Filename)%(IlcInputFolderContents.Extension) $(_Runtime_Path)%(IlcInputFolderContents.Filename)%(IlcInputFolderContents.Extension)" />
       <TestCommandLines Include="copy /y $(_TestILCFolder)\default.rd.xml  %EXECUTION_DIR%" />
-      <TestCommandLines Include="call $(_TestILCFolder)\ilc.exe -ExeName xunit.console.netcore.exe -in %EXECUTION_DIR% -out %EXECUTION_DIR%\native -usedefaultpinvoke -buildtype $(ILCBuildType) -v diag" />
-      <TestCommandLines Include="set ILCERRORLEVEL=%ERRORLEVEL%" />
-      <TestCommandLines Include="if NOT [%ILCERRORLEVEL%] == [0] exit /b %ILCERRORLEVEL%" />
+      <TestCommandLines Include="robocopy /MOV /NP %EXECUTION_DIR% %EXECUTION_DIR%\ilc.input /XF %~nx0" />
+      <TestCommandLines Include="setlocal enabledelayedexpansion" />
```

Comment:
The "/MOV" flag to robocopy makes me suspicious. RunTests.cmd is supposed to be repeatable...